dbcon/joblist/tupleaggregatestep.cpp

```diff
@@ -2754,35 +2788,71 @@ void TupleAggregateStep::prep1PhaseDistinctAggregate(
             scaleAggSub.push_back(scaleProj[j]);
             precisionAggSub.push_back(precisionProj[j]);
             typeAggSub.push_back(typeProj[j]);
-            csNumAggSub.push_back(csNumProj[j]);
+            csNumAggSub.push_back(csNumProj[i]);
```

Comment:
Could you please explain why `csNumProj[j]` became `csNumProj[i]` here? I'm a little confused.